src/edu/ucsd/library/xdre/tab/TabularRecord.java

```diff
@@ -494,7 +495,7 @@ private void addSubject( Map<String,String> data, Element e, String header,
             addAttribute( el, "parseType", rdfNS, "Collection" );
             if ( element == null ) { element = type; }
 
-            if ( header.endsWith("scientific name") || header.endsWith("common name") 
+            if ( header.endsWith("scientific name") || header.endsWith("common name") || header.endsWith("anatomy") 
```

Comment:
It sounds good. Thanks Matt. I've refactored the related subject codes so that we only need to add one line of code for a new subject, see commit https://github.com/ucsdlib/damsmanager/commit/52cb0d7c64d9f87af362cf474ff7cb6dc699084d.